shared/router-v2/tab-bar/container.js

```diff
@@ -1,17 +1,21 @@
 // @flow
 import * as Tabs from '../../constants/tabs'
+import * as ConfigGen from '../../actions/config-gen'
 import * as ProfileGen from '../../actions/profile-gen'
 import * as RouteTreeGen from '../../actions/route-tree-gen'
+import * as TrackerConstants from '../../constants/tracker2'
 import TabBar from '.'
 import {connect} from '../../util/container'
 import {memoize} from '../../util/memoize'
+import openURL from '../../util/open-url'
 
 type OwnProps = {|
   selectedTab: Tabs.Tab,
 |}
 
 const mapStateToProps = state => ({
   _badgeNumbers: state.notifications.navBadges,
+  fullname: TrackerConstants.getDetails(state, state.config.username).fullname || '',
```

Comment:
don't need `|| ''`

Reply:
this is needed actually as the type in detail is nullable

Reply:
ah ok :+1: